homeassistant/helpers/config_validation.py

```diff
@@ -81,14 +82,20 @@ def validate(obj: Dict) -> Dict:
 
 def boolean(value: Any) -> bool:
     """Validate and coerce a boolean value."""
+    if value is None:
+        return False
```

Comment:
Should we allow this? I think this could be quite unintuitive.

Reply:
On the face of it I agree, however I am concerned about regressions if it's not there.

Reply:
I don't fear too many regressions, it is difficult to specify `None` in YAML, so I doubt many people do. (famous last words!)

Reply:
I suggest we remove it.

Reply:
Will remove it and see what happens :see_no_evil:

Reply:
Yeh... it didn't work there were a few tests depending on the `None` behaviour, although they were validating that it didn't actually change anything but not catching a validation exception. I've patched them so they work right.

Otherwise we can roll-back to allowing None as a value.

IDK, I'm still a bit edgy about it. Open to comments.

Reply:
Otherwise just merge it 😨